fast64_internal/f3d/f3d_writer.py

```diff
@@ -2150,7 +2113,11 @@ def saveTextureLoading(
                 ]
             )  # added in
 
+    # Tag tile size command for tile scrolling
     tileSizeCommand = DPSetTileSize(f3d.G_TX_RENDERTILE + texIndex, sl, tl, sh, th)
+    tileSizeCommand.tags = GfxTag.TileScroll0 if texIndex == 0 else GfxTag.TileScroll1
```

Comment:
```suggestion
    tileSizeCommand.tags |= GfxTag.TileScroll0 if texIndex == 0 else GfxTag.TileScroll1
```
(imagining there may be other tags at play, `=` would overwrite them)

Reply:
The default is GfxTag.NoTag, so its the desired behaviour, unless there's a way to specify no flags.

Reply:
Oh right, NoTag should be removed, you can use `GfxTag(0)` for "no flags"

Reply:
Gotcha, updated.